rocAL/rocAL/source/pipeline/master_graph.cpp

```diff
@@ -277,20 +274,13 @@ MasterGraph::create_single_graph()
 MasterGraph::Status
 MasterGraph::build()
 {
-    if(_output_images.empty())
-        THROW("No output images are there, cannot create the pipeline")
+    if(_internal_tensor_list.empty())
+        THROW("No output tensors are there, cannot create the pipeline")
 
-    // Verify all output images have the same dimension, otherwise creating a unified tensor from them is not supported
-    _output_image_info = _output_images.front()->info();
```

Comment:
Is this check not needed in tensor pipeline?

Reply:
Yes removed intentionally